bin/carmi

```diff
@@ -66,21 +66,31 @@ async function run() {
       /node_modules/,
       // collect dependencies as we run, note that this ignores `node_modules`
       // but can be moved up so its not
-      function(filepath) {
+      function (filepath) {
         collectDependencies.push(filepath)
         return false
-      },
+      }
     ],
-    envName: 'carmi',
+    envName: 'carmi'
   })
 
-  const statsFilePath = path.resolve(options.stats);
+  const statsFilePath = options.stats ?
+    path.resolve(options.stats) :
+    // default path for stats
+    getCacheFilePath({
+      fileType: 'stats',
```

Comment:
I used this to differentiate between cache files for stats and standard cache files.